javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,38 @@
+const span = document.querySelectorAll('span');
+const body = document.querySelector('body');
+const container = document.querySelector('.container');
+let characterIndex = 0;
+let typedTotal = 0;
+let typedWrong = 0;
+
+body.addEventListener('keydown', typing);
+
+function typing(e) {
+  typedTotal++;
+  if (e.key !== span[characterIndex].textContent) {
+    span[characterIndex].classList.add('wrong');
+    typedWrong++;
+  } else {
+    span[characterIndex].classList.remove('underline');
+    span[characterIndex].classList.remove('wrong');
+    span[characterIndex].classList.add('right');
+    if (characterIndex < 29) {
```

Comment:
We don't want to listen for specific numbers here, we want this to be dynamic. What if you added another span to the HTML? We should be utilizing the `length` of the nodelist here